src/env.ts

```diff
@@ -16,4 +19,7 @@ export const env: Env = {
     host: process.env.REACT_APP_API_HOST?.toString() || 'error',
     errorHelper: process.env.REACT_APP_API_ERROR_HELPER_HOST?.toString() || 'error',
   },
+  google: {
+    login: process.env.REACT_APP_CLIENT_ID?.toString() || 'error',
+  },
```

Comment:
이런식으로 이름 명확하게 지어주면 좋을것같습니다

```suggestion
  google: {
    clientId: process.env.REACT_APP_GOOGLE_CLIENT_ID?.toString() || 'error',
  },
```